api/libproject/devstart.rb

```diff
@@ -456,6 +456,19 @@ def run_bigquery_tests(cmd_name, *args)
   :fn => ->(*args) { run_bigquery_tests("bigquerytest", *args) }
 })
 
+def run_rainforest_tests(cmd_name, *args)
+  ensure_docker cmd_name, args
+  common = Common.new
+  token = `gsutil cat gs://all-of-us-rw-staging-credentials/rainforest-key.txt`
+  common.run_inline %W{rainforest run all --token #{token}}
```

Comment:
Doesn't this need to be parameterized to run a particular test suite or in a particular environment?

Reply:
It may need to, I want to talk to Alex Minor about our strategy around this. Right now we can just run in staging, so environment can't be parameterized.